backend/uv-backend.py

```diff
@@ -409,6 +433,395 @@ def credit_card_details(transactionId):
+            ship_bol_number = details['TRANSACTION'][0]['SHIP.BOLNUMBER']
+            audit_flag = ''
+            if (ship_bol_number):
+                for x in range(0, 1):
+                    if(details['TRANSACTION'][x]['SHIP.BOLNUMBER']):
+                        order_detail['commentLabel'] = (
+                            details['TRANSACTION'][x]['SHIP.BOLNUMBER'])
+            void_reason = details['TRANSACTION'][0]['SHIP.ADDR_MV'][0]['SHIP.ADDR']
+            if(void_reason):
+                if('AUDIT.FLAG' in details['TRANSACTION'][0].keys()):
+                    audit_flag = details['TRANSACTION'][x]['AUDIT.FLAG']
+                    audit_flag = 'AUDIT FLAG: '+audit_flag
+                    return_count = details['TRANSACTION'][0]['RETURN.TRANS_MV'][0]['RETURN.TRANS']
+                    if (audit_flag.count("Return ") > 1 or audit_flag.count("XCHG ") > 1):
+                        audit_flag = "RETURN REASON: "+str(audit_flag)
+                    else:
+                        audit_flag = 'VOID REASON: '+audit_flag
+            coupons = details['TRANSACTION'][0]['GEN.COUPON.ID_MV'][0]['GEN.COUPON.ID']
+            if(coupons):
+                order_detail['coupons'] = 'GENERATED COUPON(S): '+coupons
+            if('AUDIT.FLAG' in details['TRANSACTION'][0].keys() and details['TRANSACTION'][0]['AUDIT.FLAG']):
+                order_detail['auditFlag'] = 'AUDIT FLAG: '+audit_flag
+                return_count = details['TRANSACTION'][0]['RETURN.TRANS_MV'][0]['RETURN.TRANS']
+                for i in range(len(return_count)):
+                    order_detail['returnIds'] = details['TRANSACTION'][0]['SHIP.ADDR_MV'][x]['SHIP.ADDR']
+            if('ERCPT.EMAIL' in details['TRANSACTION'][0].keys()):
+                email = details['TRANSACTION'][0]['ERCPT.EMAIL']
+                order_detail['email'] = 'ERCPT TO: '+email
+            response = {
+                "orderDetail": order_detail,
+            }
+            return Response(
+                json.dumps(response),
+                status=200,
+                mimetype='application/json'
+            )
+        else:
+            msg = 'Order number not found'
+            data = {
+                'msg': msg
+            }
+            return Response(
+                json.dumps(data),
+                status=404,
+                mimetype='application/json'
+            )
+    else:
+        msg = 'Transaction file not found'
+        data = {
+            'msg': msg
+        }
+        return Response(
+            json.dumps(data),
+            status=404,
+            mimetype='application/json'
+        )
+        
+#############################################################
+###################### Discount API #########################
+#############################################################
+
+@app.route('/transaction/discount/<transactionId>', methods=['GET'])
```

Comment:
Routes should be [RESTful](https://medium.com/@thejasonfile/restful-routing-2056f799223e)